build.gradle

```diff
@@ -47,7 +47,7 @@ install4j {
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '2.14.1'
+    gradleVersion = '3.0'
```

Comment:
This is not necessary anymore. See https://gradle.org/blog/gradle-3-0-m1-unleash-the-daemon/

`./gradlew wrapper --gradle-version 3.0` has the same effect as `./gradlew wrapper` with that config.

I'll just change it directly in the master branch.